stolostron/backlog#23648 mod sno create (#3688)
open-cluster-management/rhacm-docs: clusters/create_cluster_on_prem.adoc

You can use the {product-title} console to create an on-premises {ocp} cluster. This process is recommended instead of the bare metal process that is deprecated.

> It is best practice for you to use this procedure for creating a single-node OpenShift (SNO) cluster. You can create a single-node OpenShift cluster on VMware vSphere, Red Hat OpenStack, Red Hat Virtualization Platform, and in a bare metal environment. A single-node OpenShift cluster contains only a single node which hosts the control plane services and the user workloads. This configuration can be helpful when you want to minimize the resource footprint of the cluster. 

Reviewer: Add a comma in the following statement: A single-node OpenShift cluster contains only a single node, which hosts the control plane services and the user workloads.

> It is best practice for you to use this procedure for creating a single-node OpenShift (SNO) cluster. You can create a single-node OpenShift cluster on VMware vSphere, Red Hat OpenStack, Red Hat Virtualization Platform, and in a bare metal environment. A single-node OpenShift cluster contains only a single node, which hosts the control plane services and the user workloads. This configuration can be helpful when you want to minimize the resource footprint of the cluster. 

Reviewer 2: >You can create a single-node OpenShift cluster on VMware vSphere, Red Hat OpenStack, Red Hat Virtualization Platform, and in a bare metal environment. 

It's worth mentioning that even though you can deploy single node on all of these platforms, it will be deployed with absolutely no platform integration ("none platform")

Reviewer 3: To be explicit, `platform=none`

If there is already a base DNS domain that is associated with the selected credential that you configured for your provider account, that value is populated in that field. You can change the value by overwriting it, but this setting cannot be changed after the cluster is created. The base domain of your provider is used to create routes to your {ocp} cluster components. It is configured in the DNS of your cluster provider as a Start of Authority (SOA) record. 

The *OpenShift version* identifies the version of the {ocp-short} image that is used to create the cluster. If the version that you want to use is available, you can select the image from the list of images. If the image that you want to use is not a standard image, you can enter the URL to the image that you want to use. See xref:../clusters/release_images.adoc#release-images[Release images] for more information about release images. 

> When you select an OpenShift version that is 4.9 or later, an option to select *Install single node OpenShift (SNO)* is displayed. A single-node OpenShift cluster contains a single node which hosts the control plane services and the user workloads.

Reviewer 2: If these docs are for 2.5, it's probably worth mentioning that adding additional nodes to a single node cluster is not supported, as that is only supported starting ACM 2.6 and OCP 4.11

> If you want your cluster to be a single-node OpenShift cluster, select the single-node OpenShift option. If you are creating a single-node OpenShift cluster, add the `spec:skipMachinePools` parameter in the YAML content with a value of `true` by completing the following steps:

Reviewer 2: >add the `spec:skipMachinePools` parameter in the YAML content with a value of `true` by completing the following steps

Which YAML is this referring to? I've never heard about such requirement. Could you give me some more details about this?

Reviewer 3: I have no idea. Suggest we remove it.

When you select an OpenShift version that is 4.9 or later, an option to select *Install single node OpenShift (SNO)* is displayed. A single-node OpenShift cluster contains a single node which hosts the control plane services and the user workloads. You cannot add additional nodes to a single-node OpenShift cluster after it is created. 

If you want your cluster to be a single-node OpenShift cluster, select the single-node OpenShift option. 

> *Note:* The single-node OpenShift control plane requires 4 additional CPU cores than a control plane node requires for a multinode control plane cluster. For example, the single-node OpenShift cluster requires 8 cores while a multinode cluster requires 4 cores.  

Reviewer 2: The +4 thing is not a "rule" and is not true in general, it just currently happens to be 4 more in SNO. So perhaps change the phrasing to something like:

```suggestion
*Note:* The single-node OpenShift control plane requires 8 CPU cores as opposed to a control plane node for a multinode control plane cluster which only requires 4 CPU cores.
```

Author: Got it. Content updated. Thanks @omertuc .

+
*Note:* This value must match the name that you used to create the DNS records listed in the prerequisites section. If not provided, the DNS must be pre-configured so that `test.apps.` resolves correctly.

You can view the status of the installation on the _Clusters_ navigation page.

Continue with xref:../clusters/access_cluster.adoc#accessing-your-cluster[Accessing your cluster] for instructions for accessing your cluster. 

> [#dynamic-metrics-for-sno]

Reviewer: As discussed through Slack

"I agree with your suggestion to add Dynamic metrics for SNO clusters section to the  Observing environments documentation, It might be best to add the section after this section: https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/observability/observing-environments-intro#external-metric-query "

Author: Thanks @dockerymick. Moved to suggested location.
